clients/pkg/promtail/targets/cloudflare/fields_test.go

```diff
@@ -0,0 +1,49 @@
+package cloudflare
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFields(t *testing.T) {
+	tests := []struct {
+		name             string
+		fieldsType       FieldsType
+		additionalFields []string
+		expected         []string
+	}{
+		{
+			name:             "Default fields",
+			fieldsType:       FieldsTypeDefault,
```

Comment:
```suggestion
			fieldsType:       "default",
```
Nit: if you use the string as literal, we'll be also testing for accidental change to the constant, which in this case I think makes sense as it's part of user input.